android/shopify_buy_plugin/src/main/java/com/shopify/unity/buy/WebIntentFragment.java

```diff
@@ -0,0 +1,41 @@
+package com.shopify.unity.buy;
+
+import android.app.Fragment;
+import android.net.Uri;
+import android.support.customtabs.CustomTabsIntent;
+
+public class WebIntentFragment extends Fragment {
+
+    private boolean didShowIntent = false;
```

Comment:
Did you try https://developer.android.com/reference/android/support/customtabs/CustomTabsCallback.html#onNavigationEvent(int, android.os.Bundle)
```
int TAB_HIDDEN
Sent when the tab becomes hidden.
```